fastlane/Fastfile

```diff
@@ -119,7 +119,7 @@ lane :open_pr_upgrading_dependencies do |options|
       head: "#{branch}",
       base: "main",
       body: "Upgrade iOS to #{new_ios_version} and Android to #{new_android_version}",
-      labels: ["build"]
+      labels: ["dependencies"]
```

Comment:
Is `dependencies` in the list of supported tags from `danger`? I think we should add it there as well as in the plugin, for the changelog.

Reply:
Oh you're right, I was pretty sure it was one of the supported labels lol